src/Keyboard_pt_BR.h

```diff
@@ -0,0 +1,44 @@
+/*
+  Keyboard_pt_PT.h
+
+  Copyright (c) 2022, Edgar Bonet
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef KEYBOARD_PT_BR_h
+#define KEYBOARD_PT_BR_h
+
+#include "HID.h"
+
+#if !defined(_USING_HID)
+
+#warning "Using legacy HID core (non pluggable)"
+
+#else
+
+//================================================================================
+//================================================================================
+//  Keyboard
+
+// pt_BR keys
+#define KEY_MASCULINE_ORDINAL    (136+0x31)
+#define KEY_SIX                  (136+0x23)
```

Comment:
This two can be removed. The purpose of these macros is to give access to keys that would not be otherwise accessible. These two can be accessed as `']'` and `'6'`.

Only `KEY_ACUTE`, `KEY_C_CEDILLA` and `KEY_TILDE` need to be defined here.